Lib/test/test_os.py

```diff
@@ -735,6 +737,85 @@ def test_stat_result_pickle(self):
                 unpickled = pickle.loads(p)
                 self.assertEqual(result, unpickled)
 
+    def check_statx_attributes(self, fname):
+        maximal_mask = 0
+        for name in dir(os):
+            if name.startswith('STATX_'):
```

Comment:
feels like there should be a better way to group / gather these (maybe have them all live as constants in a Python class that acts as a namespace?)

Reply:
Similar constants for other `os` functions are also dumped in the module namespace.  That doesn't mean we can't do something different, but we'd be setting somewhat of a precedent.

(It would be nice to have an `IntFlag` enum class that can report which members are valid given a bitmask, but adding an import dependency to `os` doesn't seem like a good idea.)